components/raftstore/src/store/peer.rs

```diff
@@ -1193,12 +1193,6 @@ impl Peer {
 
         self.add_ready_metric(&ready, &mut ctx.raft_metrics.ready);
 
-        if !ready.committed_entries.as_ref().map_or(true, Vec::is_empty)
```

Comment:
Why delete this?

Reply:
Because we will assign a new time  to `current_time` even if it is not `None`.